static/xref/script.js

```diff
@@ -158,6 +158,15 @@ async function ready() {
   const newTypesElement = createInput("types", allTypes);
   document.querySelector("input[name='types']").replaceWith(newTypesElement);
 
+  const termsList = document.createElement("datalist");
```

Comment:
This is going to be awesome... just a nit tho: why not reuse the existing #termlist element instead of replacing it? You could instead use `DocumentFragment` to collect the new `option` elements, and then just replace just the children of termlist.

Reply:
Ah. Will do that!